docs/manual/manual_underactuated_torque_correction.rst

```diff
@@ -1,7 +1,7 @@
 .. include:: ../text_colors.rst
 .. toctree::
 
-.. |propeller_name| replace:: UP12
+.. |propeller_name| replace:: UPF-23-12
```

Comment:
Changing the name like this guy says: https://app.asana.com/1/64765879788601/project/1203746501506412/task/1209169070583569